gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Mar 20 18:09:37 CET 2017
+#Sat Apr 29 11:24:25 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
```

Comment:
Just curious because I've seen it in two Gradle upgrade PRs now...  Why does TripleA use the `all` distro instead of the `bin` distro?  Will something in the Travis build break without the extra artifacts contained in the `all` distro?